render-ws-java-client/src/main/java/org/janelia/render/client/newsolver/solvers/affine/AffineAlignBlockWorker.java

```diff
@@ -463,8 +464,8 @@ private static <M extends Model<M> & Affine2D<M>, S extends Model<S> & Affine2D<
 			for (final Tile<M> groupedTile : groupedTiles) {
 				final List<Tile<M>> imageTiles = solveItem.groupedTileToTiles().get(groupedTile);
 
-				if (groupedTiles.size() > 1)
-					LOG.info("assignStabilizingModel: z={} grouped tile [{}] contains {} image tiles.",
+				if (groupedTiles.size() > 1 && LOG.isDebugEnabled())
```

Comment:
This should also probably be `imageTiles.size() > 0` (and remove `isDebugEnabled`).